api/representation_test.go

```diff
@@ -293,28 +303,38 @@ func TestRequest_oapigenTaskFromConfigTask(t *testing.T) {
 					Services: &oapigen.ServicesModuleInput{
 						Regexp: config.String("^api$"),
 					},
+					ConsulKv: &oapigen.ConsulKVModuleInput{
+						Path:       "fake-path",
+						Recurse:    config.Bool(false),
+						Datacenter: config.String("dc"),
+						Namespace:  config.String("ns"),
+					},
 				},
 			},
 		},
 		{
-			name: "with_consul_kv_module_input",
+			name: "with_module_input_services_names",
```

Comment:
not sure if the diff makes this clear - I combined consul-kv test-case with "with_module_inputs" test case. I added an extra test case for services names since this test case is missing and cannot be combined with "with_module_inputs"